.tekton/release-service-catalog-periodic-pull-request.yaml

```diff
@@ -0,0 +1,596 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/konflux-ci/release-service-catalog?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/cancel-in-progress: "true"
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "staging-periodic"
```

Comment:
See this here. I still believe we should be able to just add these in the development branch, provided this here says "staging" and the target branch should also be hardcoded to "staging" probably (and hardcoded to "development" branch in the existing one).

Reply:
It's already in the tekton config files of development branch.

Reply:
What I'm talking about is staging configs in the development branch. We don't have them there today. I am saying we should be able to add them there.